sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -27,6 +27,11 @@
 _image_df = common.read_catalog('gcp/images.csv',
                                 pull_frequency_hours=_PULL_FREQUENCY_HOURS)
 
+if _image_df[_image_df['Tag'] == 'skypilot:gpu-debian-11'].empty:
```

Comment:
Q: do we expect doing this for future image tags? Or can we have some auto update mechanism in the future?

Reply:
Good point! I changed it to automatically refresh when the tag is failed to find in the catalog. Wdyt?